### ISSUE TITLE
HealthCheckResult is not exported

### ISSUE THREAD

**Issue by TrejGun** [nestjs/terminus#639]
unable to add proper types to `readiness` because `HealthCheckResult` is not exported

```
@Controller('health')
export class HealthController {
  constructor(
    private health: HealthCheckService,
    private db: TypeOrmHealthIndicator,
  ) {}

  @Get()
  @HealthCheck()
  readiness(): Promise<HealthCheckResult> {
    return this.health.check([
      () => this.db.pingCheck('database', { timeout: 300 }),
    ]);
  }
}
```

also example is broken. it should be just `() =>` not `() => async () =>`

**Comment by TrejGun** [comment]
also deprecation message refers to 
https://docs.nestjs.com/migration-guide#terminus
but there is nothing about terminus

**Comment by BrunnerLivio** [comment]
@TrejGun thanks for submitting this issue. Indeed the HealthCheckResult has not been properly exported - will fix asap. The address works now as well!

**Comment by TrejGun** [comment]
thanks

**Comment by BrunnerLivio** [comment]
Fixed in 7.0.1
